src/vllm_tgis_adapter/grpc/adapters.py

```diff
@@ -126,12 +145,36 @@ async def validate_adapters(
                 "num_virtual_tokens", 0
             ),
         )
-        return {"prompt_adapter_request": prompt_adapter_request}
+    return {"prompt_adapter_request": prompt_adapter_request}
 
     # All other types unsupported
     TGISValidationError.AdapterUnsupported.error(adapter_metadata.adapter_type)  # noqa: RET503
 
 
+async def _load_lora_adapter(
+    request: SingleGenerationRequest
+    | BatchedGenerationRequest
+    | BatchedTokenizeRequest,
+    adapter_id: str,
+    adapter_metadata: AdapterMetadata,
+    vllm_model_handler: OpenAIServingModels,
+) -> LoRARequest:
+    load_request = LoadLoRAAdapterRequest(
+        lora_path=adapter_metadata.full_path,
+        lora_name=adapter_id,
+    )
+    load_result = await vllm_model_handler.load_lora_adapter(
+        request=load_request,
+        base_model_name=request.model_id,
+    )
+    if isinstance(load_result, ErrorResponse):
+        raise TypeError(load_result.message)
```

Comment:
If we make this a `ValueError` instead then the exception handling in the server will ensure that the user gets a `StatusCode.INVALID_ARGUMENT` type response with this message, which would be helpful for them to know what went wrong

Reply:
so it was originally a ValueError, but our linter was complaining and wanted to make it a value error instead, how do i make an exception for the linter?

Reply:
The rules for ruff can all be found here: https://docs.astral.sh/ruff/rules/

They can be turned off individually by adding a comment at the end of the line like `## noqa: ${code}`

In this case you can see the error is coming from [tryceratops](https://docs.astral.sh/ruff/rules/#tryceratops-try) and you can disable with `## noqa: TRY004`